src/zcbenchmarks.cpp

```diff
@@ -88,3 +90,15 @@ double benchmark_create_joinsplit()
     assert(pourtx.Verify(*pzerocashParams));
     return ret;
 }
+
+double benchmark_solve_equihash()
+{
+    const char *testing = "testing";
```

Comment:
I think we should have both ``benchmark_mine_block`` and ``benchmark_equihash_solution_attempt``.